src/components/Comment.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+class Comment extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = { hello: "Hello", world: "World!" };
+    this.sayHelloAll = this.sayHelloAll.bind(this);
+    this.sayHelloWorld = this.sayHelloWorld.bind(this);
+  }
+
+  sayHelloAll() {
+    this.setState({ world: "All!" });
+  }
+
+  sayHelloWorld() {
+    this.setState({ world: "World!" });
+  }
+
+  render() {
+    const style = {
```

Comment:
it would be better to connect comment.less/css/sass/scss file to your component and move these styles there

Reply:
For example, I transferred the style to the App.css style file and imported the style. As I understand it, if a style is set as an object and used in a component, it is not visible to other components, it doesn't allow reuse. Thanks for your note.

Reply:
it's ok to have styles in js (css-in-js approach), but not in render :)

IMO it's not really critical for this task as it's just set up

Reply:
I got it. I have read in the documentation about several ways to style. Thanks.